README.md

```diff
@@ -25,25 +25,25 @@ Usage
 You can use the lower level nvml bindings
 
 ```python
->>> from pynvml import *
->>> nvmlInit()
+>>> from pynvml.nvml import *
+>>> nvml_init()
```

Comment:
Did these actually change in our version of the code?  Or were these docstrings just very out of date?

Reply:
Yeah - We did change the structure and the name of the initialization and finalization function wrappers.  Since we left all the other function-wrapper names unchanged, it might make sense to leave `nvmlInit()` and `nvmlShutdown()` alone for now (but all the names should probably be changed eventually)